js-exercises/minima/minima.js

```diff
@@ -0,0 +1,28 @@
+const isNumber = (num) => typeof num === 'number' && !Number.isNaN(num);
+
+function minima(numberOfElementsToReturn, array) {
+  const isValidNumber = isNumber(numberOfElementsToReturn);
+
+  if (!isValidNumber) {
+    throw new TypeError(`expected number , got ${typeof numberOfElementsToReturn}`);
+  }
+
+  if (numberOfElementsToReturn < 0) {
+    throw new Error('length must be greater or equal to zero.');
+  }
+
+  if (!Array.isArray(array)) {
+    throw new TypeError(`expected array , got ${typeof array}`);
+  }
+
+  const inputArray = array.filter(element => isNumber(element));
```

Comment:
```suggestion
  const inputArray = array.filter(isNumber);
```

Reply:
```suggestion
  const inputArray = array.every(isNumber);
```

Reply:
@ArfatSalman Array.prototype.every returns boolean right. but we are expecting an array and we are sorting that array. I think filtering is correct solution. Correct me if I am wrong.
```suggestion
  const inputArray = array.filter(isNumber);
```